edx_exams/apps/api/v1/views.py

```diff
@@ -400,10 +414,28 @@ class ExamAttemptView(ExamsAPIView):
 
     PUT Response Values
         {'exam_attempt_id': <attempt_id>}: The attempt id of the attempt being updated
+        
+    **Exceptions**
+        * HTTP_400_BAD_REQUEST
+        * HTTP_403_FORBIDDEN
     """
 
     authentication_classes = (JwtAuthentication,)
 
+    def get(self, request, attempt_id):
```

Comment:
I actually think we don't want the attempt ID to be a path parameter, and instead this endpoint should be returning any active attempt (meaning the attempt has started but not been submitted yet). 

If we do need to fetch an attempt by its ID, maybe we should make the `attempt_id` a query parameter instead?